apps/api/src/controllers/email-verification.controller.ts

```diff
@@ -0,0 +1,75 @@
+import { NextFunction, Request, Response } from "express";
+
+import { config } from "@/config/index.js";
+import { ServiceRegistry } from "@/services/index.js";
+
+export function makeEmailVerificationController(services: ServiceRegistry) {
+  /**
+   * Email Verification Controller
+   * Handles email verification endpoints
+   */
+  return {
+    /**
+     * Verify an email verification token
+     * @param req Request object
+     * @param res Response object
+     * @param next Next function
+     */
+    async verifyEmail(req: Request, res: Response, next: NextFunction) {
```

Comment:
Handler for `/verify-email` endpoint. The result of this is a redirect to `/profile/profile?success=false&message=${message}`. Let me know if this is ok for now, or we need adjustments.

Reply:
makes sense. created a UI feature request to match this: https://github.com/recallnet/js-recall/issues/772